src/main/target/OMNIBUSF4FW/target.h

```diff
@@ -25,16 +25,25 @@
 // OMNIBUSF4FW for Version 2
 // OMNIBUSF4FW1/OFW1 for Public Test Version
 // (Not a valid target, .mk file must be supplied by a user)
+// OMNIBUSF4V6
 
-#if defined(OMNIBUSF4FW)
-#define TARGET_BOARD_IDENTIFIER "OBFW"
-#define USBD_PRODUCT_STRING "OmnibusF4 Fireworks"
+#if defined(OMNIBUSF4V6)
+#define TARGET_BOARD_IDENTIFIER "OBV6"
 #elif defined(OMNIBUSF4FW1)
 #define TARGET_BOARD_IDENTIFIER "OFW1"
+#else 
+#define TARGET_BOARD_IDENTIFIER "OBFW"
+#endif
+
+#if defined(OMNIBUSF4FW1)
 #define USBD_PRODUCT_STRING "OmnibusF4 FWProto1"
+#elif defined(OMNIBUSF4V6)
+#define USBD_PRODUCT_STRING "OmnibusF4 V6"
+#else
+#define USBD_PRODUCT_STRING "OmnibusF4 Fireworks"
 #endif
 
-#if defined(OMNIBUSF4FW)
+#if defined(OMNIBUSF4FW) || defined(OMNIBUSF4V6)
```

Comment:
This won't do what you think it does - `OMNIBUSF4FW` is always defined, since it is the base target.

Reply:
corrected!